packages/devtools/src/launcher.js

```diff
@@ -79,7 +79,7 @@ async function launchChrome (capabilities) {
 
     const chrome = await launchChromeBrowser({
         chromePath: chromeOptions.binary,
-        ignoreDefaultArgs,
+        ignoreDefaultFlags: true,
         chromeFlags
```

Comment:
@koggdal Can you also simplify the condition [here](https://github.com/koggdal/webdriverio/blob/devtools-fix-ignore-chrome-flags/packages/devtools/src/launcher.js#L55)?

Reply:
That line should be unaffected by these changes. Could you clarify what you'd like me to simplify? The line you link to should still have the same logic, as we need to still support the filtering logic, and the boolean value. The line you commented on here changes because that's the API of `chrome-launcher` and we're doing all the filtering and default handling here before passing it to `chrome-launcher`.

Reply:
Got it thanks.